http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/leonardoServiceContractModel.scala

```diff
@@ -0,0 +1,71 @@
+package org.broadinstitute.dsde.workbench.leonardo.service
```

Comment:
I like this class -- should it go in `model` instead of `service`?

Reply:
These contract specific models don't need to be shared outside service, so I kinda prefer they stay in `service` and maybe we can even make them package private 🤔